openstf/exceptions.py

```diff
@@ -0,0 +1,31 @@
+"""Define custom exception classes.
```

Comment:
I think you need to add some license info here in order to pass all the build checks